nomad/local/deploy_generator.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
We should add a `BUILD` file to `nomad/local` with the following contents:

```
shell_library()
```

This will ensure that Pants will keep these scripts formatted and linted like all our other scripts.

Reply:
Related, I assume that `nomad` has some facility for formatting it's HCL files, similar to how `packer` does.

It would be good to put that into the `Makefile`, and get it wired up into our verify pipeline.

Reply:
consider a `set -euo pipefail` which is, in general, a decent base bash setup (no unset variables, bail out if something exits nonzero, etc)